sdk/communication/azure-communication-email/swagger/SWAGGER.md

```diff
@@ -15,9 +15,9 @@ autorest SWAGGER.md
 
 ### Settings
 ``` yaml
-package-version: 1.0.0
-tag: package-2024-07-01-preview
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e64ad693df24b47d4009eece6663c8d95cf94be6/specification/communication/data-plane/Email/readme.md
+package-version: 1.1.0
```

Comment:
Not sure why we have two different copies of the swagger codegen settings? Which one is getting used?

Reply:
It should be the SWAGGER.md - I'm not sure why the README.md was in there.